src/ingest_validation_tools/validation_utils.py

```diff
@@ -196,6 +196,8 @@ def get_tsv_errors(
     offline: bool = False,
     encoding: str = "utf-8",
     ignore_deprecation: bool = False,
+    report_type: ReportType = ReportType.STR,
+    cedar_api_key: str = "",
```

Comment:
Flagging addition of cedar_api_key that would need to be passed to get_tsv_errors where it's called in ingest-api.